src/utils.cpp

```diff
@@ -693,6 +693,29 @@ std::vector<std::pair<std::string, std::string>> get_cgroup_paths(
   return result;
 }
 
+bool is_module_loaded(const std::string &module)
+{
+  if (module == "vmlinux")
+  {
+    return true;
+  }
```

Comment:
Even though I hate this kernel style convention. I think we don't use brackets for single statements under conditionals
```suggestion
  if (module == "vmlinux")
    return true;
```

Reply:
Ah, OK, I've got no issue with changing this. Didn't know that `clang-format` didn't support this (but a quick Google reveals it's a relatively new feature).

Reply:
@matthew-olson-intel Actually you can ignore my comment. Seems like bpftrace doesn't require the removing of these brackets - at least according to @viktormalik

Reply:
All right -- either way!